ts/screens/profile/ProfileMainScreen2.tsx

```diff
@@ -0,0 +1,152 @@
+  profileNameSurnameSelector,
+  profileFiscalCodeSelector
+} from "../../store/reducers/profile";
+import NameSurnameIcon from "../../../img/assistance/nameSurname.svg";
+import FiscalCodeIcon from "../../../img/assistance/fiscalCode.svg";
+import EmailIcon from "../../../img/assistance/email.svg";
+
+type Props = ReturnType<typeof mapStateToProps>;
+
+const mapStateToProps = (state: GlobalState) => ({
+  sessionToken: isLoggedIn(state.authentication)
+    ? state.authentication.sessionToken
+    : undefined,
+  walletToken: isLoggedInWithSessionInfo(state.authentication)
+    ? state.authentication.sessionInfo.walletToken
+    : undefined,
+  profileEmail: profileEmailSelector(state),
+  isEmailValidated: isProfileEmailValidatedSelector(state),
+  hasProfileEmail: hasProfileEmailSelector(state),
+  nameSurname: profileNameSurnameSelector(state),
+  fiscalCode: profileFiscalCodeSelector(state)
+});
+
+export type ContextualHelpPropsMarkdown = {
+  title: TranslationKeys;
+  body: TranslationKeys;
+};
+
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "profile.main.contextualHelpTitle",
+  body: "profile.main.contextualHelpContent2"
+};
+
+const ProfileMainScreen2 = (props: Props) => {
+  const title = I18n.t("profile.main.title");
+  return (
+    <BaseScreenComponent
+      goBack={true}
+      headerTitle={title}
+      accessibilityLabel={title}
+      contextualHelpMarkdown={contextualHelpMarkdown}
+    >
+      <ScreenContent title={title} >
+        {screenContent(props)}
+      </ScreenContent>
+    </BaseScreenComponent>
+  );
+};
+
+function screenContent(props: Props) {
+  const { nameSurname, profileEmail, fiscalCode } = props;
+  const style = StyleSheet.create({
+    list: {
+      marginTop: 8
+    }
+  });
+  return <List
+    style={style.list}
+    withContentLateralPadding>
+    {/* Show name and surname */}
+    {nameSurname && (
+      <ProfileListComponent
+        Icon={NameSurnameIcon}
+        title={I18n.t("profile.data.list.nameSurname")}
+        subTitle={nameSurname}
+        testID="name-surname" />
+    )}
+    {/* Show fiscal code */}
+    {fiscalCode && (<ProfileListComponent
+      Icon={FiscalCodeIcon}
+      title={I18n.t("profile.data.list.fiscalCode")}
+      subTitle={fiscalCode}
+      testID="fical-code" />
+    )}
+    {/* Show email */}
+    {profileEmail && (<ProfileListComponent
+      Icon={EmailIcon}
+      title={I18n.t("profile.data.list.email")}
+      subTitle={pipe(
+        profileEmail,
+        O.getOrElse(() => I18n.t("global.remoteStates.notAvailable"))
+      )}
+      testID="email" />
+    )}
+  </List>;
+}
+
+// TODO: manage profile icons and study flex (see if there is a standard IO Theme
+export const ProfileListComponent = (props: {
+  Icon?: React.FC<SvgProps>;
+  title: string;
+  subTitle: string | undefined;
+  testID: string;
+}) => {
+  const { Icon, title, subTitle, testID } = props;
+  const iconProps = { width: 28, height: "auto" };
```

Comment:
same as the style component this kind of attributes can be declared outside of the component function